### PR TITLE
community: update for compatibility with Meilisearch v1.8

### PR DESCRIPTION
**Pull request by CaroFG**
Thank you for contributing to LangChain!

- [x] **PR title**: "package: description"
  - Where "package" is whichever of langchain, community, core, experimental, etc. is being modified. Use "docs: ..." for purely docs changes, "templates: ..." for template changes, "infra: ..." for CI changes.
  - Example: "community: add foobar LLM"

- [x] **PR message**: ***Delete this entire checklist*** and replace with
    - **Description:** Updates Meilisearch vectorstore for compatibility with v1.8. Adds [”showRankingScore”: true”](https://www.meilisearch.com/docs/reference/api/search#ranking-score) in the search parameters and replaces `_semanticScore` field with ` _rankingScore`

- [ ] **Add tests and docs**: If you're adding a new integration, please include
  1. a test for the integration, preferably unit tests that do not rely on network access,
  2. an example notebook showing its use. It lives in `docs/docs/integrations` directory.

- [x] **Lint and test**: Run `make format`, `make lint` and `make test` from the root of the package(s) you've modified. See contribution guidelines for more: https://python.langchain.com/docs/contributing/

Additional guidelines:
- Make sure optional dependencies are imported within a function.
- Please do not add dependencies to pyproject.toml files (even optional ones) unless they are required for unit tests.
- Most PRs should not touch more than one package.
- Changes should be backwards compatible.
- If you are adding something to community, do not re-import it in langchain.

If no one reviews your PR within a few days, please @-mention one of baskaryan, efriis, eyurtsev, ccurme, vbarda, hwchase17.

### CODE REVIEW

**Comment by baskaryan** [comment]
out of curiosity will this break compatibility with meilsearch < v1.8? if so should we document this somewhere and add a nice error message?

**Comment by CaroFG** [comment]
Hello,

With Meilisearch v1.8, the `_semanticScore` field is no longer returned in the search response. Its value is identical to `_rankingScore`, which can be obtained by setting `"showRankingScore": true` in the search parameters.

In Meilisearch v1.7, adding `"showRankingScore": true` in the search parameters will return both the `_semanticScore` and `_rankingScore` fields in the search response.

In conclusion, these changes should not be breaking for users working with Meilisearch v1.7.
